stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/options/message/internal/MessageOptionsView.kt

```diff
@@ -200,10 +198,9 @@ internal class MessageOptionsView : FrameLayout {
     }
 
     private fun configureBlock(configuration: Configuration, style: MessageListViewStyle) {
-        if (configuration.blockEnabled) {
-            binding.blockTV.configureListItem(style.messageOptionsText, style.replyIcon)
-        } else {
-            binding.blockTV.isVisible = false
```

Comment:
How to make `blockTV` invisible now?

Reply:
`binding.blockTV.isVisible = configuration.blockEnabled`

Reply:
I focused too much in the deleted lines and took little in the added ones in this review 😵‍💫